sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -140,7 +141,7 @@ private[hive] object SparkSQLCLIDriver extends Logging {
       // Hadoop-20 and above - we need to augment classpath using hiveconf
       // components.
       // See also: code in ExecDriver.java
-      var loader = conf.getClassLoader
+      var loader = orginalClassLoader
       val auxJars = HiveConf.getVar(conf, HiveConf.ConfVars.HIVEAUXJARS)
       if (StringUtils.isNotBlank(auxJars)) {
```

Comment:
Please add another test case to cover `Utilities.addToClassPath(loader, StringUtils.split(auxJars, ","))`.

Reply:
I see already test case  is present for this `test("Support hive.aux.jars.path") ` . Do you want me to add one more combination with --jars and --conf both ?

Reply:
Yes. Thank you.

Reply:
> I see already test case is present for this `test("Support hive.aux.jars.path") ` . Do you want me to add one more combination with --jars and --conf both ?

is the test case written for -- conf

Reply:
yes